extensions-contrib/materialized-view-maintenance/src/main/java/org/apache/druid/indexing/materializedview/MaterializedViewSupervisor.java

```diff
@@ -346,7 +346,8 @@ void checkSegmentsAndSubmitTasks()
           && !toBuildInterval.get(interval).equals(runningVersion.get(interval))
       ) {
         if (taskMaster.getTaskQueue().isPresent()) {
-          taskMaster.getTaskQueue().get().shutdown(runningTasks.get(interval).getId());
+          taskMaster.getTaskQueue().get().shutdown(
+              runningTasks.get(interval).getId(), "version mismatch");
```

Comment:
There is no need to break this down into a new line.

Reply:
Thanks, fixed.